cirq/linalg/states.py

```diff
@@ -21,13 +21,15 @@
 def one_hot(*,
             index: Union[None, int, Sequence[int]] = None,
             shape: Union[int, Sequence[int]],
+            value: int = 1,
             dtype: Type[np.number]) -> np.ndarray:
-    """Returns a numpy array with a single 1 entry, and 0 everywhere else.
+    """Returns a numpy array with all 0s and a single non-zero entry(default 1).
 
     Args:
         index: The index that should store 1 instead of 0.
```

Comment:
```suggestion
        index: The index that should store the 'value' argument instead of 0.
```

Reply:
Use backticks \`value\` instead of single-quotes 'value', so it is understood as a code reference `value`.